_config.yml

```diff
@@ -49,14 +49,14 @@ contentful:
     query: fields.category=common
     map:
       content_block: content
-  page:
-    query: "fields.collections[exists]=false"
   podcast:
     query: "fields.exclude_from_crossroads[ne]=true"
   episode:
     query: "content_type=episode&fields.podcast.sys.contentType.sys.id=podcast&fields.podcast.fields.exclude_from_crossroads[ne]=true"
   video:
     query: "fields.exclude_from_crossroads[ne]=true"
+    map:
+      date: published_at
```

Comment:
Was this intentional?

Reply:
IIRC this would prevent videos without a date in the past from rendering on the front end, but I'm not sure why that's relevant to this work.